content/docs/examples/multicluster/gateways/index.md

```diff
@@ -99,6 +99,13 @@ running in a second cluster.
     `httpbin.bar.global` on *any port* to be routed to the endpoint
     `<IPofCluster2IngressGateway>:15443` over an mTLS connection.
 
+    If your cluster2 Kubernetes cluster is running in an environment that does not
+    support external load-balancers, you must use the IP and tls nodePort of a node
```

Comment:
use the IP and nodePort corresponding to port 15443 of a node

Reply:
ack